### ISSUE TITLE
Wrong spawn container - Center Redwoods

### ISSUE THREAD

**Issue by Damienlee69** [thommcgrath/Beacon#276]
Just spent half the evening troubleshooting why nothing would spawn in the redwoods on the center. Turns out Beacon and the wikis are using the wrong container. Both are using lost islands version, not the islands.
Using redwoods_lostisland_c, should be redwoods_c.

**Comment by thommcgrath** [comment]
This is because both the wiki and Beacon get their data from the data mining. According to the game's files, Lost Island is correct. See https://raw.githubusercontent.com/arkutils/Obelisk/master/data/wiki/TheCenter/npc_spawns.json

**Comment by Damienlee69** [comment]
I should let you know, this was confusing for a rented PS4 server. Testing in PC it seems to work though. On Oct 13, 2023 9:15 PM, Thom McGrath ***@***.***> wrote:
This is because both the wiki and Beacon get their data from the data mining. According to the game's files, Lost Island is correct. See https://raw.githubusercontent.com/arkutils/Obelisk/master/data/wiki/TheCenter/npc_spawns.json

**Comment by thommcgrath** [comment]
Ah so console and PC are different in this case. Well... shoot. There's really nothing I can do about that. I don't have a mechanism for platform-specific map compatibility.

**Comment by Damienlee69** [comment]
No way to add an "*" note to it..a popup? This is also from just one persons testing. I have not tested any other maps either. ASE is dead to many anyways, however me and my friend did notice there seems to be a lot more rented servers now and you might have the same issue when center is released for ASA. The code may very well be quite different too.. Who knows. Are you gonna pull a WC and charge people again? Lol. On Oct 14, 2023 7:39 AM, Thom McGrath ***@***.***> wrote:
Ah so console and PC are different in this case. Well... shoot. There's really nothing I can do about that. I don't have a mechanism for platform-specific map compatibility.

**Comment by thommcgrath** [comment]
No, not really. There are options I could explore long term, but the root issue is that Beacon projects are inherently not platform specific. So how do I know which spawn points are correct for your map? So that would need to change, which is a big deal. Then I'd need to add an extensible compatibility system that can support any number of platforms. Right now, compatibility is just tracked with a single number. So making this work right is a **big** problem. As far as a note goes, it wouldn't really help anything as working around it is rather difficult. Since ASE is at the end of its life, I'm not going to put any effort into this.

For ASA though, we just don't know yet, and we won't until release. If I need to expand how compatibility is tracked for ASA, so be it. My pricing plan for Beacon is to charge by the game. I'm expanding beyond just Ark games, so it seems more fair to charge a lower price for each game than a larger price for all games. Based on what we know of ASA, it'll be different enough from ASE that it will be a separate game in Beacon, so it will be a separate purchase. But there will be upgrade pricing for anybody that already owns Omni for ASE.

**Comment by Damienlee69** [comment]
Pricing sounds fair. Especially for the seperate game model. I get that you wouldn't want to put effort into a "dying" game. Was essentially just an FYI. For the note, I just figured a little tooltip, like "if on a console rented server change this to that in your ini". I sure hope they don't screw up ASA for customizing.. Maybe they'll make it easier!! Lol. Take care.On Oct 14, 2023 1:16 PM, Thom McGrath ***@***.***> wrote:
No, not really. There are options I could explore long term, but the root issue is that Beacon projects are inherently not platform specific. So how do I know which spawn points are correct for your map? So that would need to change, which is a big deal. Then I'd need to add an extensible compatibility system that can support any number of platforms. Right now, compatibility is just tracked with a single number. So making this work right is a big problem. As far as a note goes, it wouldn't really help anything as working around it is rather difficult. Since ASE is at the end of its life, I'm not going to put any effort into this.
For ASA though, we just don't know yet, and we won't until release. If I need to expand how compatibility is tracked for ASA, so be it. My pricing plan for Beacon is to charge by the game. I'm expanding beyond just Ark games, so it seems more fair to charge a lower price for each game than a larger price for all games. Based on what we know of ASA, it'll be different enough from ASE that it will be a separate game in Beacon, so it will be a separate purchase. But there will be upgrade pricing for anybody that already owns Omni for ASE.
